content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -0,0 +1,325 @@
+    "deploy": "sls deploy --stage local"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "serverless": "^3.25.0",
+    "serverless-deployment-bucket": "^1.6.0",
+    "serverless-localstack": "^1.0.1"
+  }
+}
+```
+
+You can give the above setup a run by deploying your Serverless project and checking the Lambda function that has been created via our existing Serverless project:
+
+{{< command >}}
+$ npm run deploy
+$ awslocal lambda list-functions
+
+{
+    "Functions": [
+        {
+            "FunctionName": "serverless-elb-local-hello",
+            "FunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:serverless-elb-local-hello",
+            "Runtime": "nodejs12.x",
+            "Role": "arn:aws:iam::000000000000:role/serverless-elb-local-us-east-1-lambdaRole",
+            "Handler": "handler.hello",
+            ...
+        }
+    ]
+}
+{{< / command >}}
+
+We can now create our target Lambda functions to work with Node.js runtime and configure ELB Application Load Balancers via our `serverless.yml` configuration.
+
+## Create Lambda functions & ELB Application Load Balancers
+
+Let us now create two Lambda functions, named `hello1` and `hello2`, which can run over Node.js 12.x runtime. Open `handler.js` and replace the existing code with the following:
+
+```js
+'use strict';
+
+module.exports.hello1 = async (event) => {
+  console.log(event);
+  return {
+    "isBase64Encoded": false,
+    "statusCode": 200,
+    "statusDescription": "200 OK",
+    "headers": {
+        "Content-Type": "application/json"
+    },
+    "body": "Hello 1"
+  };
+};
+
+module.exports.hello2 = async (event) => {
+  console.log(event);
+    return {
+    "isBase64Encoded": false,
+    "statusCode": 200,
+    "statusDescription": "200 OK",
+    "headers": {
+        "Content-Type": "application/json"
+    },
+    "body": "Hello 2"
+  };
+};
+```
+
+In the above code, we have created `hello1` and `hello2` Lambda functions and created a response body for them, including the Base64 encoding status, status code, and headers. If you wish to include binary content in the response body, set the `isBase64Encoded` property to `true`. The load balancer requires retrieving the binary content to send it in the body of an HTTP response.
+
+Let us now move to the `serverless.yml` file and specify our deployment bucket and the functions we want to deploy. Our initial configuration in the `serverless.yml` file should look like this:
+
+```yaml
+service: serverless-elb
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  deploymentBucket:
+    name: testbucket
+
+functions:
+  hello1:
+    handler: handler.hello1
+    events:
+    - alb:
+        listenerArn: !Ref HTTPListener
+        priority: 1
+        conditions:
+          path: /hello1
+  hello2:
+    handler: handler.hello2
+    events:
+    - alb:
+        listenerArn: !Ref HTTPListener
+        priority: 2
+        conditions:
+          path: /hello2
+```
```

Comment:
nit: maybe we can add 3 dots `...` here to indicate that some other parts are still missing; alternatively, we could add the `plugins` and `custom` section in here as well, to make it self-contained?